lib/openapi_first/request_validation/request_body_validator.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'body_parser'
+
+module OpenapiFirst
+  module RequestValidation
+    class RequestBodyValidator
+      def initialize(operation, env)
+        @operation = operation
+        @env = env
```

Comment:
what do you think about passing only required data instead of the whole env? e.g.
```ruby
def initialize(operation, body, content_type = "application/json")
```

passing `env` around seems... like a little bit too much information passed when compared to the actual data being used. Also, the interface is less explicit

Reply:
Sounds good!
But be aware that RequestBodyValidator is not part of the "public" API. I would like to align request and response validation a little and remove some Openapi 3 jargon like "operation" to this one will probably resolve in something like a body validator or content validation ([contentType + contentSchema](https://github.com/OAI/moonwalk#requests))